### ISSUE TITLE
Remove unnecessary pandoc from setup

### ISSUE THREAD

**Issue by mmaney** [komuw/sewer#179]
### What is it that you would like to propose to add/remove/change?

setup.py uses pandoc to convert README.md to restructure text for PyPi.  According to PyPi, this should be unnecessary, so that business in setup.py and pandoc as a dependency can be removed.  Oh, and the mention of pandoc in CONTRIBUTING.

### Why do you want to add/remove/change that?

One should strive to have as few dependencies as possible, but no fewer.  Einstein said something like that.  :-)

### How do you want to go about adding/removing/changing that?

I can PR & merge it if there's not something I'm overlooking that makes pandoc actually necessary.  My guess is that long ago, PyPi didn't accept github-style markdown, hence pandoc in setup.py.

**Comment by mmaney** [comment]
I don't know the history of the pandoc conversion in setup.py, but it is certainly unnecessary with Pypi now (2020) provided you also give it the correct content type option.

**Comment by komuw** [comment]
Hi, @mmaney  
pandoc was used to read the README.md and convert it into rst so that it can be rendered correctly in pypi;

<img width="1032" alt="Screenshot 2020-08-15 at 11 06 22" src="https://user-images.githubusercontent.com/5163857/90308328-5ef2bf00-dee7-11ea-89dc-c028df6c12b2.png">

**Comment by mmaney** [comment]
Sorry, I closed this without updating the history.  It looks good now (sewer_test on testpypi), though it could have been any combination of a simpler README (perhaps using less markdown features), server-side improvements in the rendering, or the addition of an explicit content_type in setup.py to guide the rendering.

And if that isn't a complete success after all, I'll just make it into README.rst - I did a little experimenting at one point and GH seemed perfectly happy with that.  Now that so much of it has been split off into pages it's not a big deal one way or the other.
